docs/docs/02-Installation/06-debuntu.md

```diff
@@ -0,0 +1,76 @@
+# Debian 12/Ubuntu 24.04
+
+:::warning
+This script is a stripped-down version of those found in the [Proxmox Community Scripts](https://github.com/community-scripts/PromoxVE) repo. It has been adapted to work on baremetal Debian 12 or Ubuntu 24.04 installs **only**. Any other use is not supported and you use this script at your own risk.
+:::
+
+### Requirements
+
+- **Debian 12** (Buster) or
+- **Ubuntu 24.04** (Noble Numbat)
+
+The script will download and install all dependencies (except for Ollama), install Hoarder, do a basic configuration of Hoarder and Meilisearch (the search app used by Hoarder), and create and enable the systemd service files needed to run Hoarder on startup. Hoarder and Meilisearch are run in the context of their low-privilege user environments for more security.
+
+The script functions as an update script in addition to an installer. See **[Updating](#updating)**.
+
+### 1. Download the script from the [Hoarder repository](https://github.com/hoarder-app/hoarder/blob/main/hoarder-linux.sh).
+
+```
+wget https://raw.githubusercontent.com/hoarder-app/hoarder/main/hoarder-linux.sh
+```
+
+### 2. Make the script executable
+
+    ```shell
+    chmod +x ./hoarder-linux.sh
+    ```
+
+### 3. Run the script
+
+> This script must be run as `root`, or as a user with `sudo` privileges.
+
+    If this is a fresh install, then run the installer by using the following command:
+
+    ```shell
+    ./hoarder-linux.sh install
+    ```
```

Comment:
Given that this is a one time script, can we just do `bash hoarder-linux.sh install` instead of first making it executable?

Reply:
Yes that works and removes the additional instruction; I'll amend the docs to remove the chmod bit.